about.php

```diff
@@ -0,0 +1,211 @@
+<?php include'head.php';?>
+
+<div id="pcourses">
+    <span>About us</span>
+</div>
+   <h1>Our Staff</h1>
+    <div id="susanne">
+        
+    <img src="images/susanne.jpg"> 
+    <h2>Susanne Pertl</h2>
+    <p>Susanne Pertl is an executive board member of the Stern Stewart Institute e.V. and the founder of Lycee Schorge 
+	in Koudougou. Her vision for the BIT is to provide state-of-the-art education for the most committed and enthusiastic
+	young women and men of Burkina Faso. BIT  strives for a striking impact in educating entrepreneurs who are willing to 
+	create change. She aims to build future generations of young leaders and professionals in Burkina faso. 
+    More entrepreneurs lead to more opportunity, more economic growth, and more prosperity for everyone.
+        </p>
+    </div>
+    <div id="admin">
+        
+    
+        <ul>
+            
+            <li id="li">
+                <img src="images/9K5A0361.jpg"> 
+				<h2 style="position:absolute; left:20%;bottom:1px">Dr.Kabore Rodrigue</h2>
+               <p class="dop-down show">Rodrigue W. Rodrigue has 10 years of experience working in Burkina Faso. He worked for private entreprise for 2 years and spent 8 years working for the government. Recently, he is having been the director of promotion of the digital economy. He has how accepted the responsibilities of this current role a Dean of BIT. He has been teaching in higher education for over 4 years with a focus on information and communication technologies. He received his master's degree in 2013 and his doctor little in 2018. to him, innovative technologies occupy a vital place in students straining, especially focusing on entrepreneurship</p>
+            </li> 
+            <li id="li1">
+                <img src="images/9K5A0354.jpg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px">Inza Gnanou</h2>
+               <p class="dop-down1 show1">

```

Comment:
It looks like you should use id instead of class here. 

`id` means: You only have the name once in the whole HTML file
`class` means: You reuse certain styling or behavior within an HTML file and hence the class appears more often.